config/brakeman.ignore

```diff
@@ -1,23 +1,23 @@
 {
   "ignored_warnings": [
     {
-      "warning_type": "Mass Assignment",
-      "warning_code": 105,
-      "fingerprint": "17848eaedea3f609c289b9ab3468c74935be427a3c2b6444235479eb0ef77272",
-      "check_name": "PermitAttributes",
-      "message": "Potentially dangerous key allowed for mass assignment",
-      "file": "app/controllers/users_controller.rb",
-      "line": 42,
-      "link": "https://brakemanscanner.org/docs/warning_types/mass_assignment/",
-      "code": "params.require(:user).permit(:name, :email, :password, :mobile_number, :role)",
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "024ae0d4e7586e89ac6b9039fccc4a99055fc20b620d35c2efcb76b883abf998",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "app/services/planning_application_search.rb",
+      "line": 30,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "planning_applications.select(\"*, ts_rank(#{tsvector}, #{tsquery}) AS rank\").where(\"#{tsvector} @@ #{tsquery}\")",
```

Comment:
I'm using `sanitize_sql_array` on the search query. I think the issue might just be because it's buried away in a private helper method and Brakeman hasn't managed to chase it down 🤔

Reply:
hmm this seems like a thing we shouldn't ignore so i wonder why it doesn't see `sanitize_sql_array`.

Are we confident that we aren't risk to sql injection, wonder if someone's pulled up an issue on brakeman regarding this

Reply:
Have you tried to make the helper method public? Does Brakeman still raise this issue?

Reply:
It still raises it if I just do

```
def records_matching_description
    planning_applications
      .select("*, ts_rank(to_tsvector('english', description), to_tsquery('english', 'banana | orange')) AS rank")
      .where("to_tsvector('english', #{kate}) @@ to_tsquery('english', 'banana | orange')")
      .order(rank: :desc)
  end

  def kate
    "description"
  end
```

🤯

Reply:
I've refactored it so that brakeman is happy!

Still not sure if the problem was 1. brakeman needlessly freaks out at any use of `#{}` in a sql statement or 2. brakeman was rightfully freaking out because I was passing `sanitize_sql_array` isn't as effective if you only hand it an sql fragment instead of the whole statement.

Anyway, I will sleep better at night knowing brakeman is happy 🤣